docs/docs/client.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Pomerium TCP Clients
+description: Learn how to create secured TCP connections to services behind Pomerium.
+---
+
+# Pomerium Desktop and CLI Clients
+
+Pomerium is capable of creating secure connections to services like SSH, Redis, and more by creating a TCP tunnel to the service with a local client. This article describes configuring a route to accept TCP connections, and using either the CLI or GUI client to connect to it.
+
+## Create a TCP Route
+
+1. Specify this new Route as a TCP Route by prefixing `tcp+` in the **From** field, along with a port suffix.
+
+   ::: tip
+   The port is not used to connect to the Pomerium proxy service from the internet; this will always be port 443 (unless otherwise defined in `config.yaml`). Rather, the port defined in **From** is part of the mapping to the individual route. In this way, you can create multiple routes that share a DNS entry, differentiated by the port to determine which route they use.
```

Comment:
Might be long winded for a `tip` section but should an example be included for clarity?  My go-to is always accessing SSH and PSQL on the same host without requiring different DNS entires.

Reply:
We can do that, and we'll just make it part of the step and not a tip.

Reply:
How does 08cc640be32739917861f2b7cfd87f8001201ad3 grab ya?

Reply:
👍